### ISSUE TITLE
Installing gem files due to HTTP 404

### ISSUE THREAD

**Issue by ericgoedtel** [jhuckabee/dynamodb-local#2]
`gem install dynamodb-local` results in:

```
Building native extensions.  This could take a while...

▽
ERROR:  Error installing dynamodb-local-0.0.5.gem:
	ERROR: Failed to build gem native extension.

    current directory: /var/lib/gems/2.3.0/gems/dynamodb-local-0.0.5
/usr/bin/ruby2.3 -rubygems /var/lib/gems/2.3.0/gems/rake-11.3.0/exe/rake RUBYARCHDIR=/var/lib/gems/2.3.0/extensions/x86_64-linux/2.3.0/dynamodb-local-0.0.5 RUBYLIBDIR=/var/lib/gems/2.3.0/extensions/x86_64-linux/2.3.0/dynamodb-local-0.0.5
fatal: Not a git repository (or any of the parent directories): .git
mkdir -p lib/dynamodb/local/ext
Downloading DynamoDB local...rake aborted!
OpenURI::HTTPError: 404 Not Found
/var/lib/gems/2.3.0/gems/dynamodb-local-0.0.5/Rakefile:21:in `block (2 levels) in <top (required)>'

▽
/var/lib/gems/2.3.0/gems/dynamodb-local-0.0.5/Rakefile:20:in `open'
/var/lib/gems/2.3.0/gems/dynamodb-local-0.0.5/Rakefile:20:in `block in <top (required)>'
/var/lib/gems/2.3.0/gems/rake-11.3.0/exe/rake:27:in `<main>'
Tasks: TOP => default => prepare
(See full trace by running task with --trace)

rake failed, exit code 1

Gem files will remain installed in /var/lib/gems/2.3.0/gems/dynamodb-local-0.0.5 for inspection.
Results logged to /var/lib/gems/2.3.0/extensions/x86_64-linux/2.3.0/dynamodb-local-0.0.5/gem_make.out
```

Inspection of the rake file indicates that it is looking for an AWS S3 bucket:
`http://dynamodb-local.s3-website-us-west-2.amazonaws.com/dynamodb_local_latest`

Trying to fetch anything from this bucket results in an HTTP 404. It looks like the bucket may no longer exist or the object within it may no longer exist.

**Comment by s32x** [comment]
In case you haven't solved your issue yet I've recreated the gem with the new bucket. Replacing `dynamodb-local` with `local_dynamodb` in your Gemfile should resolve this.

**Comment by ericgoedtel** [comment]
Thanks for taking the time! I ended up going with the default AWS installation instead of using a Gem but that will definitely be useful to others.

**Comment by jhuckabee** [comment]
Fixed in 0.0.6.
